README.md

```diff
@@ -103,3 +103,114 @@ There are many ways to [contribute](https://github.com/Microsoft/AdaptiveCards/b
 
 This project has adopted the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/). For more information see 
 the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.
+
+## Supported Features
+|Feature|Schema Version|Platform|Parsing|Rendering|
+|---|---|---|:---:|:---:|
+|Action.OpenUrl|v1.0|All| :o: | :o:|
```

Comment:
nit: Is it possible to change the color of the circles or use check marks instead? I think that would improve readability.